data_steward/tools/add_hpo.py

```diff
@@ -363,6 +382,16 @@ def main(hpo_id, org_id, hpo_name, bucket_name, display_order, addition_type,
                         '--bucket_name',
                         required=True,
                         help='Name of the GCS bucket')
+    parser.add_argument(
+        '-s',
+        '--us_state',
+        required=True,
+        # type=lambda x: True if x.startswith("PIIState_") else,
```

Comment:
Try defining a separate function for this, and in that function raise a Value Error if the input does not start with `PIIState_`

Reply:
sure.